controllers/velero.go

```diff
@@ -357,6 +361,16 @@ func (r *DPAReconciler) customizeVeleroDeployment(dpa *oadpv1alpha1.DataProtecti
 		veleroContainer.Args = append(veleroContainer.Args, fmt.Sprintf("--resource-timeout=%v", resourceTimeoutString))
 	}
 
+	// check for default-snapshot-move-data parameter
+	if len(getSnapshotMoveDataValue(dpa)) > 0 {
```

Comment:
create variable prior to the `if`s to avoid calling function twice?

Reply:
Done, updated